[#25][Bug] Fix: README completeness and typos (#32)
nimblehq/react-templates: README.md

---

<p align="center">
    <a href="https://www.npmjs.com/package/cra-template-nimble"><img src="https://badgen.net/npm/v/cra-template-nimble" /></a>
    <a href="https://www.npmjs.com/package/cra-template-nimble"><img src="https://badgen.net/npm/dy/cra-template-nimble" /></a>
</p>

> Our templates offer a rich boilerplate to jump-start React-based application development for [Create React App](https://github.com/facebook/create-react-app).

Reviewer: Not sure but should we remove the `s` here as this is only one template 🤔 

```suggestion
Our template offers a rich boilerplate to jump-start React-based application development for [Create React App](https://github.com/facebook/create-react-app).
```

Author: ✅ c063f6f ;-)

> Our template offers a rich boilerplate to jump-start React-based application development for [Create React App](https://github.com/facebook/create-react-app).

Reviewer 2: wondering if it should be `with [Create React App]` or `from [Create React App]` instead of `for` 🤔  I think we are extending what the create-react-app offers, so I thought using "for" might be slightly odd 😛

Author: Definitely not `for` ^^
Both with and from seems good, but I would prefer "with" (it sounds like cra stands next to the project, not just at the origin — which is the case as cra is not ejected and remains a project dependency).,

Author: ✅ 0ac66df

## Getting Started

### Prerequisites

[![node-version-image][node-v-src]](https://nodejs.org/download/release/v14.18.1/)

### Usage

> To use this template, add `--template nimble` when creating a new app from the `create-react-app` package.

Reviewer 2: not sure if it should be called package 🤔 I feel like a 'command'
as a whole when we add the template params

```suggestion
To use this template, add `--template nimble` when creating a new app from the `create-react-app` command.
```

Author: True, the package name is `react-scripts` (which contains the config associated with the cra command).
And in the context of "when creating a new app", it definitely refers to the command ;-)

Author: ✅ 9687b2c

└── template.json
```

`Typescript` is used by default for our React applications.

> To the standard files from a non-ejected `create-react-app` project, this template adds a folder structure in `/src` that follows our [React Convention](https://nimblehq.co/compass/development/code-conventions/javascript/react/#project-structure).

Reviewer 2: I just re-read this again and the previous version so I felt that the newer one might not very well converted on the meaning 😅 just with 1 word

```suggestion
With the standard files from a non-ejected `create-react-app` project, this template adds a folder structure in `/src` that follows our [React Convention](https://nimblehq.co/compass/development/code-conventions/javascript/react/#project-structure).
```

Author: ✅ 9687b2c ;-)

└── template.json
```

> `Typescript` is used by default for our React applications. To the standard files from a non-ejected `create-react-app` project, this templates ads a folder structure in `/src` that follows our

Reviewer 3: Micro typoe:

```suggestion
`Typescript` is used by default for our React applications. To the standard files from a non-ejected `create-react-app` project, this templates adds a folder structure in `/src` that follows our
```

Author: ✅ c063f6f ;-)

> `Typescript` is used by default for our React applications. To the standard files from a non-ejected `create-react-app` project, this templates adds a folder structure in `/src` that follows our
> [React Convention](https://nimblehq.co/compass/development/code-conventions/javascript/react/#project-structure).

Reviewer 2: I think the line break is a little bit odd, maybe you can consider:

Having a soft line break between the first 2 sentences: (also fix the `template` in singular form)
```suggestion
`Typescript` is used by default for our React applications.  
To the standard files from a non-ejected `create-react-app` project, this template adds a folder structure in `/src` that follows our [React Convention](https://nimblehq.co/compass/development/code-conventions/javascript/react/#project-structure).
```

or just inlining:

```suggestion
`Typescript` is used by default for our React applications. To the standard files from a non-ejected `create-react-app` project, this template adds a folder structure in `/src` that follows our [React Convention](https://nimblehq.co/compass/development/code-conventions/javascript/react/#project-structure).
```

Author: ✅ 0ac66df

I've found another bad line break (I think it comes from the auto-linter of some IDE... like auto line break after XX chars).

Should be all fixed now ;-) Thanks!
